src/test/scala/scalismo/faces/momo/MoMoTests.scala

```diff
@@ -162,7 +167,7 @@ class MoMoTests extends FacesTestSuite {
       norm(meanCoeffs.shape) + norm(meanCoeffs.color) should be < 0.01 * momoCoeffs.shape.length + 0.01 * momoCoeffs.color.length
     }
 
-    val f = File.createTempFile("momo", ".h5")
+    val f = File.createTempFile("momo", ".json")
```

Comment:
I usually call the files `.h5.json` but this works as well.

Reply:
That is a good idea - thx.